Production/govuk_ios/ViewModels/HomeViewModel.swift

```diff
@@ -6,17 +6,34 @@ struct HomeViewModel {
 
     let configService: AppConfigServiceInterface
     let searchButtonPrimaryAction: (() -> Void)?
+    let recentActivityAction: (() -> Void)?
     var widgets: [WidgetView] {
         [
-            searchWidget
+            searchWidget,
+            recentlyViewedWidget
         ].compactMap { $0 }
     }
+    private var recentlyViewedWidget: WidgetView? {
```

Comment:
This is incredibly hard to press on the home page

Reply:
i have now made whole widget clickable